.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,69 @@
+---
+name: E2E Tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  config:
+    env:
+      TEST_CMD: gotestsum --junitfile e2e-tests.xml --format standard-verbose --
+      UNIT_TAGS: unit
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GitHubSecurityLab/actions-permissions/monitor@v1
+        with:
+          config: ${{ vars.PERMISSIONS_CONFIG }}
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
+      - run: go install gotest.tools/gotestsum@latest
+      - run: |
+          set -Eeou pipefail
+          mkdir "$XDG_CONFIG_HOME/atlascli"
+          cat <<EOF > "$XDG_CONFIG_HOME/atlascli/config.toml"
+          [e2e]
+            org_id = "5e429e7706822c6eac4d5971"
+            public_api_key = "AAUMGJXA"
```

Comment:
can we have a var for this public key?

Reply:
this is fake data, not real, I could replace it with "test" if you like

Reply:
yes if possible to make it clear that is not a real API key